be-rust/src/commonMain/resources/lang/temper/be/rust/temper-core/Cargo.toml

```diff
@@ -4,6 +4,11 @@ version = "0.6.0"
 license = "Apache-2.0 OR MIT"
 edition = "2021"
 rust-version = "1.71.1"
+readme = false
+documentation = "https://temperlang.dev/tutorial/"
```

Comment:
I suspect a lot of crates go to docs.rs pages for this field, but I haven't reviewed that.